PacketDotNet/TcpFields.cs

```diff
@@ -29,14 +29,18 @@ public struct TcpFields
     {
 #pragma warning disable 1591
         // flag bitmasks
-        public readonly static int TCP_CWR_MASK = 0x0080;
-        public readonly static int TCP_ECN_MASK = 0x0040;
-        public readonly static int TCP_URG_MASK = 0x0020;
-        public readonly static int TCP_ACK_MASK = 0x0010;
-        public readonly static int TCP_PSH_MASK = 0x0008;
-        public readonly static int TCP_RST_MASK = 0x0004;
-        public readonly static int TCP_SYN_MASK = 0x0002;
-        public readonly static int TCP_FIN_MASK = 0x0001;
+        public readonly static int TCP_FLAGC_MASK = 0x0800;
```

Comment:
Hmm. Maybe it is related to a line ending change? I just pushed out a bunch of changes to clean up some inconsistent line endings. If you can rebase against master the extra changes might go away. Based on your comment I'd only expect to see 4 new lines added here.